.github/workflows/macos-build.yml

```diff
@@ -357,41 +476,59 @@ jobs:
         sed $SEDOPTION '/.*builtins-pgo/d' out/Default/mksnapshot_args
         sed $SEDOPTION '/--turbo-profiling-input/d' out/Default/mksnapshot_args
         sed $SEDOPTION '/The gn arg use_goma=true .*/d' out/Default/mksnapshot_args
-        if [ "$SKIP_DIST_ZIP" != "1" ]; then
```

Comment:
Noting for posterity that this is being removed as it stopped being necessary after we removed debug builds